Density Stats.ipynb

```diff
@@ -0,0 +1,155 @@
+{
+ "cells": [
```

Comment:
I don't know what this notebook is but it should not be here.

Reply:
Ok.This is jupyter notebook.